### DISCUSSION TITLE
ro-crate-metadata.json: main node (./) has to have "hasPart"

### DISCUSSION THREAD

**Discussion by SteffenBrinckmann** [TheELNConsortium/TheELNFileFormat#7] [General]
Kadi4Mat's example is nicely to parse, since it's main node (./) has a 'hasPart' entry. This entry is the head of the graph.
Can we all agree to include that, incl. elabFTW ;-)?

**Comment by NicolasCARPi** [comment]
In my examples, the main node (`@id => './'`) also `hasParts`, but in the wrong format (not in arrays with `@id`). In fact I'm currently building my import code on the Kadi4mat example ^^

EDIT: ok I see now, in kadi implementation, files don't exist as root nodes, but only as parts in hasPart of a dataset.

**Comment by SteffenBrinckmann** [comment]
I did the same, using Kadi as example for import and adopting the export to
be the same style
